### PR TITLE
Fix/fetch shared files

### PR DESCRIPTION
**Pull request by nettashviro**
fix the fetching of getSharedFIles.

### CODE REVIEW

**Review by Shahar-Y — CHANGES_REQUESTED** [review]
Please add more documentation about the change: 
what changed in the return value, and what was it before the change? And the meaning of the new fields.

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The user in context is now more enriched (with fields like job, rank, current unit...)
 - The metrics metric now contains a timestamp and more information about the user.
 
+### Fixed
+
+- Fix the fetching of GetSharedFiles bug that returns 404.
```
[the remark this answers is not in the document]
Comment (nettashviro) [reply CHANGELOG.md]: done

File: server/auth/auth.go [hunk 2]
```diff
@@ -184,6 +184,8 @@ func (r *Router) UserMiddleware(c *gin.Context, secret string, authURL string) {
 
 	c.Set(oauth.ContextAppKey, oauth.DriveAppID)
 
+	c.Set(oauth.ContextAppKey, oauth.DriveAppID)
```
Comment (Shahar-Y) [inline server/auth/auth.go:187]: Why twice?

**Review by Shahar-Y — CHANGES_REQUESTED** [review]
Other than the comment, looks great! 
After it's approved, don't merge yet

**Review by Shahar-Y — DISMISSED** [review]
Looks good, Don't merge

**Pull request closed without merging**